CONTRIBUTING.md

```diff
@@ -56,6 +56,25 @@ __EOF__
 chmod +x .git/hooks/pre-commit
 ```
 
+### Running tests
+
+The integration tests are written using Python and the
+[nose](https://nose.readthedocs.io/en/latest/index.html) testing framework.
+They are run in a docker container, so you need
+[docker](https://docs.docker.com/get-docker/) installed locally.
+
+```bash
+make run-test
+```
+
+You can filter the tests you are running with the `TEST` environment variable.
+
+```bash
+make TEST=multi run-test       # runs tests matching tests/test_multi*
+make TEST=single run-test      # runs tests _not_ matching tests/test_multi*
+make TEST=test_sample run-test # runs tests/test_sample.py
```

Comment:
Can we use an example that you can run? this one gives me `ERROR: Failure: ImportError (No module named 'test_sample')` because we don't have a test_sample file. A short one that runs in a short amount of time (about 30s on my laptop) is `make TEST=test_auth run-test`.